python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -463,6 +463,16 @@ def _db_connect(self):
             with contextlib.closing(connection.cursor()) as cursor:
                 yield (connection, cursor)
 
+    def _compute_sys_path(self):
```

Comment:
could this be something that we added to the toolkit manager instead (as a classmethod or static i guess, since it's kind of independent of the state of any instance). THinking about sphinx docs - could be nice to include it as part of manager - if nothing else so just for completeness, a dev reading the tk manager docs would be aware of the fact that you cannot grab the sys.path of core due to the magic of the swaps.